### DISCUSSION TITLE
"clicker app virus v3" requires a provisioning profile. Select a provisioning profile in the Signing & Capabilities editor.

### DISCUSSION THREAD

**Discussion by sendhelpp** [sst-inc/help#15] [Q&A - iOS] [answered]
Uhm It says I need to select a Provisioning Profile, but I don't know how to create one. It's under "Signing & Capabilities". I already selected team as none, still did not fix it. pls send help
<img width="1382" alt="Screenshot 2021-07-31 at 9 50 01 PM" src="https://user-images.githubusercontent.com/88239920/127741980-7366e5c2-ba9d-4303-8149-38b954b1435b.png">

**Comment by jiachenyee** [comment, marked as answer]
You'll need to change the run destination to an iPhone simulator. I believe you can change it to an `iPhone 12 Pro Max` simulator (@Joeispro99 @granwyntan correct me if I'm wrong) and try to run it. Refer to the annotated screenshot to find out how to change it.

![Safari -  clicker app virus v3  requires a provisioning profile  Select a provisioning profile in the Signing   Capabilities editor  · Discussion #15 · sst-inchelp  2021-07-31 at 10 12 26PM](https://user-images.githubusercontent.com/36725840/127742577-eaf8460c-ef4d-4839-b2bd-12c7b1018c7a.png)

**Reply by jiachenyee** [reply]
clarification: You should select an actual iPhone simulator, not the one that says "Any iOS Device".
![image](https://user-images.githubusercontent.com/36725840/127743951-70eb714a-4155-4a1f-8e33-682026f623f8.png)

**Reply by granwyntan** [reply]
@sendhelpp, please choose **iPhone 12 Pro Max** instead for lessons, thanks!

**Reply by sendhelpp** [reply]
it worked, thanks
